docs/source/getting_started/tutorials/sec_tutorials.rst

```diff
@@ -0,0 +1,196 @@
+.. _sec-tutorial:
+
+================================
+Spectroelectrochemistry tutorial
+================================
+
+This tutorial demonstrates importing, plotting, and exporting in-operando UV-Vis data
+as an example of spectroelectrochemistry (S-EC).
+It shows delta optical density calculation and both calculation and plotting of the full 2-D data field and
+cross sections (i.e. spectra and wavelength-vs-time).
+
+The example data is not yet publically available, thus there is no code execution shown here. 
```

Comment:
It may be publically available now! The article is under review and available in pre-print. https://www.researchsquare.com/article/rs-2605628/v1
@Caiwu-L and I will fix this up :)

Reply:
as this is not urgent, we'll wait until that article is accepted :)